uber/templates/preregistration/group_members.html

```diff
@@ -30,10 +30,36 @@
                 }
             });
         })
+
+        $("form[action='../groups/cancel_dealer']").submit(function(event){
```

Comment:
We'll have to check with our event marketplaces if we want dealers to have self-service cancellation.

Reply:
Ran this by the MAGWest chair and got the thumbs-up :+1: